superbench/config/amd_mi100_hpe.yaml

```diff
@@ -99,9 +99,31 @@ superbench:
         copy_type:
           - sm
           - dma
-    ort-inference:
```

Comment:
can't run on mi100?

Reply:
yes, ort-inference does not register on rocm platform